src/goto-symex/symex_target.h

```diff
@@ -63,21 +69,46 @@ class symex_targett
     GUARD,
   };
 
-  // read event
+  /// Reading from a shared variable \c ssa_rhs (which will thus become a
```

Comment:
:pick: Consider using imperative mood in all function docs - here `Read` instead of `Reading`. I see that further below you do use it, so would be good to keep it consistent. :grapes:

Reply:
Done.